generation/maisi/configs/config_trt.json

```diff
@@ -0,0 +1,22 @@
+{
+    "+imports": [
+        "$from monai.networks import trt_compile"
+    ],
+    "c_trt_args": {
+        "export_args": {
+            "dynamo": "$False",
+            "report": "$True"
+        },
+        "output_lists": [
+            [
+                -1
+            ],
+            [
+	    ]
+        ]
+    },
+    "device": "cuda",
+    "controlnet": "$trt_compile(@controlnet_def.to(@device), @trained_controlnet_path, @c_trt_args)",
+    "diffusion_unet": "$trt_compile(@diffusion_unet_def.to(@device), @trained_diffusion_path)",
+    "mask_generation_diffusion": "$trt_compile(@mask_generation_diffusion_def.to(@device), @trained_mask_generation_diffusion_path)"
```

Comment:
Hi @borisfom , thanks for the changes. 
Can I confirm if `mask_generation_diffusion` supports TRT? Should we also modify https://github.com/Project-MONAI/model-zoo/blob/dev/models/maisi_ct_generative/configs/inference_trt.json ?